packages/web3-eth/test/integration/subscribe.test.ts

```diff
@@ -7,17 +7,22 @@ import {
 	NewPendingTransactionsSubscription,
 	LogsSubscription,
 } from '../../src/web3_subscriptions';
+// eslint-disable-next-line
```

Comment:
You should disable the specific rule that's being violated, not completely disable the linter